src/net/sourceforge/kolmafia/session/EquipmentManager.java

```diff
@@ -924,6 +927,9 @@ public static final void addConditionalSkills(final Slot slot, AdventureResult i
         KoLCharacter.addAvailableSkill(SkillPool.SWOOP_LIKE_A_BAT);
         KoLCharacter.addAvailableSkill(SkillPool.SUMMON_CAULDRON_OF_BATS);
       }
+      case ItemPool.SHERIFF_BADGE, ItemPool.SHERIFF_PISTOL, ItemPool.SHERIFF_MOUSTACHE  -> {
+        KoLCharacter.addAvailableSkill(SkillPool.ASSERT_YOUR_AUTHORITY);
```

Comment:
this will add the skill if any is equipped. You want to add the skill only if all are equipped. see left bear arm / right bear arm.

Reply:
Adjusted this, too. Followed Bear Arms. Didn't see a comparable and condition (how do Ands work in Java again?) so I used a nested if.

Reply:
and is &&